packages/core/src/tracing/instrumentation/protocols/http2Client.js

```diff
@@ -164,6 +164,11 @@ function addHeaders(headers, span, w3cTraceContext) {
   if (!headers) {
     return;
   }
+  if (span.shouldSuppressDownstream) {
```

Comment:
I had to remind myself for a minute: `shouldSuppressDownstream` is only true if the class instance is `InstanaIgnoredSpan` AND the config is ON.

I think the readability here is not ideal.

I'd suggest to raise a PR after this PR and change the 3 occurances in our code to:

```
if (span.isIgnored AND config flag is ON)
```